File: Src/alex-pi/src/app/_api/models/web-event-log.ts
/* tslint:disable */
import { WebsiteUser } from './website-user';
export interface WebEventLog {
  id?: number;
  websiteUserId?: number;
  eventName?: string;
  doneAt?: string;
  websiteUser?: WebsiteUser;
  eventData__Copy?: string;
}

File: Src/Db.OneBase/Model/WebEventLog.cs
using System;
using System.Collections.Generic;

namespace Db.OneBase.Model
{
    public partial record WebEventLog
    {
        public int Id { get; set; }
        public int WebsiteUserId { get; set; }
        public string EventName { get; set; }
        public DateTime DoneAt { get; set; }

        public virtual WebsiteUser WebsiteUser { get; set; }
    }
}
